.mailmap

```diff
@@ -0,0 +1,13 @@
+Carlos Lancha <carlos.lancha@liferay.com> <carloslancha@users.noreply.github.com>
+Chema Balsas <jose.balsas@liferay.com> <jbalsas@gmail.com>
+Eduardo Lundgren <eduardo.lundgren@liferay.com> <eduardo.lundgren@gmail.com>
+Greg Hurrell <greg.hurrell@liferay.com> <greg@hurrell.net>
+Iván Zaera Avellón <ivan.zaera@liferay.com> <ivan@baco.local>
+Iván Zaera Avellón <ivan.zaera@liferay.com> <izaera@gmail.com>
+Iván Zaera Avellón <ivan.zaera@liferay.com> <izaera@gmx.es>
+Iván Zaera Avellón <ivan.zaera@liferay.com> <izaera@users.noreply.github.com>
```

Comment:
Not sure what to make of this... who's `ivan@baco.local`? why does they still use `gmx`? Me suspects they's outsourcing their work 🤔 

![imwatchingyou](https://user-images.githubusercontent.com/905006/95550360-0d5b4200-0a09-11eb-882f-72d5d6507863.gif)

Reply:
![i-will-find-you](https://user-images.githubusercontent.com/7074/95552382-ec94eb80-0a0c-11eb-90c1-7afe169c0330.gif)

Reply:
:scream: I got caught...

![escaping](https://media.giphy.com/media/jsHXjOW4iJm8wBH1Yz/giphy.gif)

Reply:
Sadly, I had git badly configured for some time and instead of complaining it took my local (laptop) mail address to do the commits :-(